<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyCOL</class>
 <widget class="QMainWindow" name="PyCOL">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>355</width>
    <height>555</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>355</width>
    <height>555</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PyCOL</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/menu.png</normaloff>../icons/menu.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ColorPicker">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Color Picker</string>
    </property>
   </widget>
   <widget class="QSlider" name="RSlide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Seek for Red</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="GSlide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Seek for Green</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="BSlide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Seek for Blue</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="RSet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="GSet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="BSet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ColorView">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Color Preview</string>
    </property>
   </widget>
   <widget class="QWidget" name="showColor" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>331</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border: 1px solid; border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HexDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Click to copy Hex Code</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Hex: Hex Color Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Xplore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Click here to explore more colors</string>
    </property>
    <property name="text">
     <string>Explore More Colors</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="UserInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>359</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Enter your custom color. . .</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter your custom color to apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplyBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click to apply the custom color</string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InfoBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>More details of PyCOL</string>
    </property>
    <property name="text">
     <string>More Info</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="PyMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="PyCOL_Menu">
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>PyCOL Menu</string>
    </property>
    <property name="title">
     <string>PyCOL</string>
    </property>
    <addaction name="flatUI_op"/>
    <addaction name="RndColor"/>
    <addaction name="separator"/>
    <addaction name="PyDoc"/>
    <addaction name="ExitApp"/>
   </widget>
   <addaction name="PyCOL_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="flatUI_op">
   <property name="text">
    <string>FlatUI Color</string>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="RndColor">
   <property name="text">
    <string>Random Color</string>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="PyDoc">
   <property name="text">
    <string>Application Docs</string>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="ExitApp">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
